queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon%';
```

Comment:
* Kindly use the proper query as this '%mon%' would look for the string "mon" anywhere in the names of all animals while the question says select only animals that end with "mon". Hint 💭 : this should be the right query '%mon'